src/components/SelectCard.tsx

```diff
@@ -21,42 +27,55 @@ export const SelectCard = ({
         selectedIndex === cardIndex
             ? { ...selectStyle.selectCard, ...selectStyle.selected }
             : selectStyle.selectCard;
+
+    //need this to get the line breaks to work from a string... so dumb lol
+    //https://stackoverflow.com/questions/54796368/line-break-inside-string-react-native
+    const text = displayedText.replace(/\\n/g, '\n');
     return (
         <TouchableOpacity
             activeOpacity={1}
             style={style}
             onPress={() => onPressHandler(cardIndex)}
         >
-            <Text category="s1" style={{ paddingVertical: 5 }}>
-                {displayedText}
+            <Text category="h3" style={{ paddingVertical: 5, marginRight: 15 }}>
+                {text}
             </Text>
-            {bottomRow()}
+            {icons()}
         </TouchableOpacity>
     );
 };
 const selectStyle = StyleSheet.create({
     selectCard: {
-        width: '100%',
-        height: 80,
+        width: 300,
+        height: 73,
         marginVertical: 5,
         borderRadius: 30,
         backgroundColor: 'white',
-        padding: 5,
+        padding: 10,
         alignItems: 'center',
+        justifyContent: 'center',
+        flexDirection: 'row',
 
-        shadowColor: '#000',
+        shadowColor: optionCardShadowColorNeutral,
         shadowOffset: {
             width: 0,
             height: 2,
         },
-        shadowOpacity: 0.25,
-        shadowRadius: 3.84,
+        shadowOpacity: 0.5,
+        shadowRadius: 18 / 2,
 
         elevation: 5,
     },
     selected: {
-        borderColor: 'rgb(40, 224, 224)',
-        borderWidth: 2,
-        padding: 3,
+        borderColor: optionCardBorderColor,
+        borderWidth: 3,
+        padding: 7,
+        shadowColor: optionCardShadowColorHighlighted, //hard to see this shadow on ios
+        shadowRadius: 23 / 2,
```

Comment:
11.5?

Reply:
I was following a styling guide for converting shadows from sketch (michelle's design application) to ios, and it says to divide sketch's radius numbers by 2 to get the correct effect. I figured it'd be best to leave her original numbers in there along with the division. Do you think I should just manually halve the numbers?

Reply:
nevermind, follow recommended style guide